packages/solarwinds-apm/src/index.cjs.ts

```diff
@@ -23,6 +23,12 @@ const setInit = setter("init")
 if (setInit && versionCheck()) {
   setInit()
 
+  console.warn(
+    "It looks like you're initialising solarwinds-apm using a require call or flag.",
+    "This initialisation method is not recommended and may be removed in a future release.",
+    "See https://github.com/solarwinds/apm-js/tree/main/packages/solarwinds-apm#installation-and-setup",
+  )
```

Comment:
In the future I'd like to change the project to target ESM only with a small CJS wrapper given the improvements we have seen in load times and performance from using ESM, and since it would greatly simplify the structure and maintainance.